src/ArrayHelper.php

```diff
@@ -58,7 +57,7 @@ final class ArrayHelper
      * ]
      * ```
      *
-     * @param array|object|string $object The object to be converted into an array.
+     * @param mixed $object The object to be converted into an array.
```

Comment:
?

Reply:
Code of method work with mixed value. For any non-array and non-object value returns `[$value]`.